pkg/providers/amifamily/bootstrap/bottlerocketsettings.go

```diff
@@ -48,7 +48,7 @@ type BottlerocketKubernetes struct {
 	CloudProvider                      *string                                   `toml:"cloud-provider"`
 	ClusterCertificate                 *string                                   `toml:"cluster-certificate"`
 	ClusterName                        *string                                   `toml:"cluster-name"`
-	ClusterDNSIP                       *string                                   `toml:"cluster-dns-ip,omitempty"`
+	ClusterDNSIP                       []string                                  `toml:"cluster-dns-ip,omitempty"`
```

Comment:
I think we need to expand this a bit. There's a discussion over in the original issue, but as this stands, this will break anyone who is specifying a string value in their userData for the ClusterDNSIP